File: basic.py
def init_queue():
    return {
        "collection": []
    }

def print_queue(queue):
    print(queue["collection"])

def enqueue(queue, element):
    queue["collection"].append(element)

def dequeue(queue):
    if is_empty(queue):
        return None
    return queue["collection"].pop(0)

def front(queue):
    if is_empty(queue):
        return None
    return queue["collection"][0]

def size(queue):
    return len(queue["collection"])

def is_empty(queue):
    return len(queue["collection"]) == 0

File: basic.java
import java.util.ArrayList;
import java.util.List;

public class Queue {
    private List<Object> collection;
    
    public Queue() {
        this.collection = new ArrayList<>();
    }
    
    public void enqueue(Object element) {
        collection.add(element);
    }
    
    public Object dequeue() {
        if (isEmpty()) {
            return null;
        }
        return collection.remove(0);
    }
    
    public Object front() {
        if (isEmpty()) {
            return null;
        }
        return collection.get(0);
    }
    
    public int size() {
        return collection.size();
    }
    
    public boolean isEmpty() {
        return collection.isEmpty();
    }
    
    public void print() {
        System.out.println(collection);
    }
}
